File: 백준/Bronze/10810. 공 넣기/공 넣기.c
#include <stdio.h>

int main(void) {
   int a, b, x, y, z;
   scanf("%d %d", &a, &b);

   int arr[a];
   for(int i = 0; i < a; i++){
      arr[i] = 0;
   }
   
   for(int i = 0; i < b; i++){
      scanf("%d %d %d", &x, &y, &z);
      for(int j = 0; j <= y - x; j++){
         arr[(x - 1) + j] = z;
      }
   }

   for(int i = 0; i < a; i++){
      printf("%d ", arr[i]);
   }
   
   return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
arr = []
a, b = map(int, input().split())
if (len(arr) > a):
  print("error")

for i in range(a):
  arr.append(0)

for i in range(b):
  x, y, z = map(int, input().split())
  for j in range(y - x + 1):
    arr[x - 1 + j] = z

for item in arr:
  print(item, end = ' ')